tests/components/freebox/conftest.py

```diff
@@ -1,11 +1,50 @@
 """Test helpers for Freebox."""
-from unittest.mock import patch
+from unittest.mock import AsyncMock, patch
 
 import pytest
 
+from .const import (
+    DATA_CALL_GET_CALLS_LOG,
+    DATA_CONNECTION_GET_STATUS,
+    DATA_LAN_GET_HOSTS_LIST,
+    DATA_STORAGE_GET_DISKS,
+    DATA_SYSTEM_GET_CONFIG,
+    WIFI_GET_GLOBAL_CONFIG,
+)
+
 
 @pytest.fixture(autouse=True)
 def mock_path():
     """Mock path lib."""
     with patch("homeassistant.components.freebox.router.Path"):
         yield
+
+
+@pytest.fixture(name="router")
+def mock_router():
+    """Mock a successful connection."""
+    with patch("homeassistant.components.freebox.router.Freepybox") as service_mock:
+        service_mock.return_value.open = AsyncMock()
```

Comment:
You can make your code a lot more readable by doing this:
```suggestion
        instance = service_mock.return_value
        instance.open = AsyncMock()
```

Reply:
Did it but I kinda prefer the old way because the `return_value` were aligned, and it made a bit of space.